dart-api-dl-sys/Cargo.toml

```diff
@@ -12,11 +12,11 @@ static_assertions = "1.1.0"
 
 [build-dependencies]
 bindgen = "0.59.2"
-cc = "1.0.72"
-semver = "1.0.4"
+cc = "1.0.73"
+semver = "1.0.5"
 
-# v0.1.20 is on edition 2021 which is a problem as it means rust won't parse the
+# v0.2.0 is on edition 2021 which is a problem as it means rust won't parse the
```

Comment:
Not sure about this, it explain why we have the bounds we have.

Reply:
version `0.1.20` was yanked (because of the breaking change this comment refers to), version `0.2.0` is the new breaking version that this comment refers to now.